### ISSUE TITLE
[docs version 2.5]: fix broken image in deployment doc for google cloud run

### ISSUE THREAD

**Issue by withshubh** [ToolJet/ToolJet#6318]
### Summary

- Go to the Setup -> Google Cloud Run doc: https://docs.tooljet.com/docs/2.5.0/setup/google-cloud-run
- On the step 4, you'll find the broken image

Solution:
- Go to the `docs/versioned_docs/version-2.5.0/setup/google-cloud-run.md`
- On line number 50 change the image name to `port-and-capacity-postgrest.png`

### Issue Type

Documentation bug

### The entire URL of the documentation with the issue

https://docs.tooljet.com/docs/2.5.0/setup/google-cloud-run

### If the issue is confirmed, would you be willing to submit a pull request?

No

### Code of Conduct

- [X] I agree to follow the ToolJet Code of Conduct

**Comment by tejakummarikuntla** [comment]
Thank you for your interest @tarunsamanta2k20 , assigned this issue to you.

**Comment by Bhavyaatrivedi** [comment]
Can you assign this issue to me

**Comment by withshubh** [comment]
After careful consideration, we have reached a decision to consolidate all the issues that necessitate changes in each version into a single merged issue. This resolution has been implemented with the aim of minimizing the inconvenience associated with creating separate issues or pull requests for identical bugs, as an affect this issues needs to be closed.

We appreciate everyone's efforts in identifying these issues and submitting them for consideration. We hope you understand and are sorry for the inconvenience caused, would encourage you to explore other `good first issues` and `up for grabs`.
